src/data/directory/lgu/region-i-ilocos-region.json

```
@@ -1018,19 +1010,18 @@
             "name": "RICARDO D. BALDERAS"
           },
           "vice_mayor": {
-            "name": "NELSON V. GAYO"
+            "name": "BENEDICT E. LOPENA"
```

```suggestion
            "name": "JOSE BENEDICT E. LOPENA"
```

ref: https://lgu201.dilg.gov.ph/view.php?r=01&p=055&m=48